UPGRADE.md

```diff
@@ -8,6 +8,10 @@ This file contains instructions to upgrade to a new release of Gregorio.
 
 High notes are now taken into account in vertical spacings. If you would like to come back to the old way, use `\grechangecount{additionaltopspacethreshold}{6}`. High notes also influence vertical placement of alt text. If you would like to come back to the old way, give a high value to `\grechangecount{additionaltopspacealtthreshold}{6}`.
 
+### Last line of score behavior
+
+`\grelooseness` is now `-1` by default. This should prevent orphaned syllable at end of score. If you prefer the old behavior, use `\def\grelooseness{-1}` befor including your scores.
```

Comment:
Should this not say to use `\def\grelooseness{\looseness}` for the old behavior?